### PR TITLE
Updated custom range selector for float

### PR DESCRIPTION
**Pull request by bnmajor**
Updated custom range selector to offer six decimal precision for float values.

### CODE REVIEW

File: tomviz/HistogramWidget.cxx [hunk 1]
```diff
@@ -401,6 +402,15 @@ void HistogramWidget::onCustomRangeClicked()
   double maxRange[2];
   activeDataSource->getRange(maxRange);
 
+  // Get the type of the active scalar
+  auto scalar = activeDataSource->activeScalars();
+  auto array = activeDataSource->getScalarsArray(scalar);
+  auto dataType = array->GetDataType();
+  int precision = 2;
```
Comment (cryos) [inline tomviz/HistogramWidget.cxx]: So, I am not sure I understand why we would use precision of 2 for integral types, and six for float. We also support doubles (although discourage it). I would prefer to keep it simple and offer up to 6 for all, or check for double/float offer 6 and offer 0 decimal places for integral types.

Comment (cjh1) [reply tomviz/HistogramWidget.cxx]: From the issue it sounded like we want 6, just for float ( and yes double ). 2 is the current default.

Comment (cryos) [reply tomviz/HistogramWidget.cxx]: I think the default was more of an oversight, let's use 0 as our default here (for integral types), and set it to 6 for float and double.

Comment (cjh1) [reply tomviz/HistogramWidget.cxx]: Sounds like a good plan.

**Comment by cryos** [comment]
Looks great, thanks!

**Pull request merged**